blocks/showcase/showcase.css

```
@@ -49,6 +49,13 @@
   color: var(--primary-color);
 }
 
+.showcase .icon-plus.animate {
+  animation-delay: 0.25s;
+  animation-duration: 1s;
+  animation-fill-mode: both;
+  animation-name: zoom-center;
```

This is the wrong animation? On the original pages I see a fade-in, and the animation called `fade-in` seems to be the one we want here.

---

fade-in doesn't seem to have any animation affect.

---

The fade-in works only once (first scroll-down, by design) to avoid blinking of the icons.
```suggestion
  animation-name: fade-in;
```

---

I have updated it to use fade-in